src/main/java/ca/corefacility/bioinformatics/irida/ria/web/services/AnalysesListingService.java

```diff
@@ -166,7 +166,7 @@ private JobError getFirstJobError(AnalysisSubmission submission) throws Executio
 	 * @param end   {@link Date}
 	 * @return {@link Long} milliseconds
 	 */
-	private Long getDurationInMilliseconds(Date start, Date end) {
+	public Long getDurationInMilliseconds(Date start, Date end) {
```

Comment:
We may want to refactor this method into a utilities class.

Reply:
:) Thank you for doing it